commands/package.mjs

```diff
@@ -287,15 +286,22 @@ export function getCommand() {
      */
     async function _handleUnpack(argv) {
         const dbMode = argv.nedb ? "nedb" : "classic-level";
-        const typeDir = currentPackageType.toLowerCase() + "s";
+        const typeDir = ""
+        if (!argv.outputDirectory || !argv.inputDirectory) {
+            typeDir = currentPackageType.toLowerCase() + "s";
+            if (!currentPackageId) {
+                console.error(chalk.red("No package ID is currently set. Use `package workon <id>` to set it."));
+                return;
+            }
+        }
         const compendiumName = argv.compendiumName ?? argv.value;
-        if ( !compendiumName ) {
+        if ( !compendiumName && ( dbMode === "nedb" || (!argv.outputDirectory || !argv.inputDirectory))) {
```

Comment:
We reuse `(!argv.outputDirectory || !argv.inputDirectory)` three times here, let's pull it to a variable such as `usingDefaultDirectory`

Reply:
that's simple enough.
Done